sdk/nodejs/iot/getRegistrationCode.ts

```diff
@@ -47,24 +29,6 @@ export interface GetRegistrationCodeResult {
 }
 /**
  * Gets a registration code used to register a CA certificate with AWS IoT.
```

Comment:
What's happening, curious..

Reply:
Not sure, I'll investigate why this example got dropped but it seems unrelated to the changes here.